Controller/HelperController.php

```diff
@@ -439,4 +454,35 @@ private function retrieveFieldDescription(AdminInterface $admin, $field)
 
         return $fieldDescription;
     }
+
+    /**
+     * Retrieve the filter field description given by field name.
+     *
+     * @param AdminInterface $admin
+     * @param string         $field
+     *
+     * @return \Symfony\Component\Form\FormInterface
+     *
+     * @throws \RuntimeException
+     */
+    private function retrieveFilterFieldDescription(AdminInterface $admin, $field)
+    {
+        $admin->getFilterFieldDescriptions();
+
+        $fieldDescription = $admin->getFilterFieldDescription($field);
+
+        if (!$fieldDescription) {
+            throw new \RuntimeException(sprintf('The field "%s" does not exist.', $field));
+        }
+
+        if ($fieldDescription->getType() !== 'doctrine_orm_model_autocomplete') {
```

Comment:
This is just a small check. I think that  it is not needed to add it to configuration. I have removed it to not include doctrine orm inside this universal admin.